features/provider_due_amount.feature

```diff
@@ -13,3 +13,17 @@ Examples:
 | 0               | 0             | 0          |
 
 
+
+Scenario Outline: Payment type breakdown
+Given the account has a balance of <balance>
+When payment of <due amount> is due
+Then a levy payment of <levy amount> is made
+And a government payment of <government amount> is made
+And a employer payment of <employer amount> is expected
+Examples:
+| balance | due amount | levy amount | government amount | employer amount |
```

Comment:
Broadly seems good to me. Couple of points:

1. Is it misleading to say the employer account is debited at this point? This is not strictly true
2. We should stop referring to a levy account. The current feeling is that it is an employer account. We should probably stick with levy account for now, so keep it consistent; but plan a wholesale re-wording once content clan have fully settled on the name